components/alertbanner/index.css

```diff
@@ -12,25 +12,23 @@
  */
 
 .spectrum-AlertBanner {
-	--spectrum-alert-banner-neutral-background: var(--spectrum-neutral-subdued-background-color-default);
-
 	--spectrum-alert-banner-min-height: var(--spectrum-alert-banner-minimum-height);
 	--spectrum-alert-banner-max-inline-size: var(--spectrum-alert-banner-width);
 	--spectrum-alert-banner-inline-size: auto;
 
 	--spectrum-alert-banner-font-size: var(--spectrum-font-size-100);
 	--spectrum-alert-banner-line-height: var(--spectrum-line-height-100);
 	--spectrum-alert-banner-font-family: var(--spectrum-sans-font-family-stack);
-	--spectrum-alert-banner-icon-size: var(--spectrum-workflow-icon-size-100); /* stylelint-disable-line spectrum-tools/no-unused-custom-properties -- used in passthrough */
+	--spectrum-alert-banner-icon-size: var(--spectrum-workflow-icon-size-100);
```

Comment:
Unnecessary lint ignore rule; passthroughs are accounted for by the no-unused-custom-properties plugin.